### ISSUE TITLE
[preview] RUF028 false positive in decorators.

### ISSUE THREAD

**Issue by randolf-scholz** [astral-sh/ruff#11689]
Rule RUF028 produces false positives for decorators:

https://play.ruff.rs/139a4202-caac-4f92-9662-671492adc377

```python
import pytest

@pytest.mark.parametrize(
    "test_input,expected",
     [
          ("3+5",  8 ),
          ("17+2", 19),
     ],
)  # fmt: skip
def test_eval(test_input, expected):
    assert eval(test_input) == expected
```

The `# fmt: skip` comment gets flagged as invalid, despite the fact that it does actually work (as one would expect).
